web/app.js

```diff
@@ -12,18 +12,93 @@
-           PDFOutlineViewer, PDFAttachmentViewer, OverlayManager,
-           PDFFindController, PDFFindBar, PDFViewer, PDFRenderingQueue,
-           PresentationModeState, parseQueryString, RenderingStates,
-           UNKNOWN_SCALE, DEFAULT_SCALE_VALUE, DEFAULT_URL, mozL10n */
+/* globals DEFAULT_URL, PDFBug, Stats */
 
 'use strict';
 
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define('pdfjs-web/app', ['exports', 'pdfjs-web/ui_utils',
+      'pdfjs-web/firefoxcom', 'pdfjs-web/download_manager',
+      'pdfjs-web/pdf_history', 'pdfjs-web/preferences', 'pdfjs-web/pdf_sidebar',
+      'pdfjs-web/view_history', 'pdfjs-web/pdf_thumbnail_viewer',
+      'pdfjs-web/secondary_toolbar', 'pdfjs-web/password_prompt',
+      'pdfjs-web/pdf_presentation_mode', 'pdfjs-web/pdf_document_properties',
+      'pdfjs-web/hand_tool', 'pdfjs-web/pdf_viewer',
+      'pdfjs-web/pdf_rendering_queue', 'pdfjs-web/pdf_link_service',
+      'pdfjs-web/pdf_outline_viewer', 'pdfjs-web/overlay_manager',
+      'pdfjs-web/pdf_attachment_viewer', 'pdfjs-web/pdf_find_controller',
+      'pdfjs-web/pdf_find_bar', 'pdfjs-web/mozPrintCallback_polyfill',
+      'pdfjs-web/pdfjs'],
+      factory);
+  } else if (typeof exports !== 'undefined') {
+    factory(exports, require('./ui_utils.js'), require('./firefoxcom.js'),
+      require('./download_manager.js'), require('./pdf_history.js'),
+      require('./preferences.js'), require('./pdf_sidebar.js'),
+      require('./view_history.js'), require('./pdf_thumbnail_viewer.js'),
+      require('./secondary_toolbar.js'), require('./password_prompt.js'),
+      require('./pdf_presentation_mode.js'),
+      require('./pdf_document_properties.js'), require('./hand_tool.js'),
+      require('./pdf_viewer.js'), require('./pdf_rendering_queue.js'),
+      require('./pdf_link_service.js'), require('./pdf_outline_viewer.js'),
+      require('./overlay_manager.js'), require('./pdf_attachment_viewer.js'),
+      require('./pdf_find_controller.js'), require('./pdf_find_bar.js'),
+      require('./mozPrintCallback_polyfill.js'), require('./pdfjs.js'));
+  } else {
+    factory((root.pdfjsWebApp = {}), root.pdfjsWebUIUtils,
+      root.pdfjsWebFirefoxCom, root.pdfjsWebDownloadManager,
+      root.pdfjsWebPDFHistory, root.pdfjsWebPreferences,
+      root.pdfjsWebPDFSidebar, root.pdfjsWebViewHistory,
+      root.pdfjsWebPDFThumbnailViewer, root.pdfjsWebSecondaryToolbar,
+      root.pdfjsWebPasswordPrompt, root.pdfjsWebPDFPresentationMode,
+      root.pdfjsWebPDFDocumentProperties, root.pdfjsWebHandTool,
+      root.pdfjsWebPDFViewer, root.pdfjsWebPDFRenderingQueue,
+      root.pdfjsWebPDFLinkService, root.pdfjsWebPDFOutlineViewer,
+      root.pdfjsWebOverlayManager, root.pdfjsWebPDFAttachmentViewer,
+      root.pdfjsWebPDFFindController, root.pdfjsWebPDFFindBar,
+      root.pdfjsWebMozPrintCallbackPolyfill, root.pdfjsWebPDFJS);
+  }
+}(this, function (exports, uiUtils, firefoxCom, downloadManagerLib,
```

Comment:
As I mentioned on IRC, the fact that only some of these have `Lib` at the end confused me initially.
So I'm wondering if we could append `Lib` consistently on *all* of these, to reduce possible confusion here?